lib/elasticsearch/api/response/explain_node.rb

```diff
@@ -1,11 +1,12 @@
 require "elasticsearch/api/response/renderable"
+require 'forwardable'
 
 module Elasticsearch
   module API
     module Response
       class ExplainNode
         include Renderable
-        extend Forwardable
+        extend ::Forwardable
```

Comment:
Note in Ruby 2.5 omitting the :: will perform a same folder constant lookup. + we need to require 'forwardable' explicitely. Should be retro-compatible